app/assets/stylesheets/components/_index.scss

```diff
@@ -1,5 +1,2 @@
-@import "button";
-@import "buttons";
-@import "cards";
-@import "form";
-@import "navbar";
\ No newline at end of file
+@import "_navbar";
+@import "_footer";
```

Comment:
What is this ?
You merged master into your branch but then you delete everything master has.
That's not what you're supposed to do.